### DISCUSSION TITLE
Spawn in an empty gamemode , no errors are shown.

### DISCUSSION THREAD

**Discussion by Cyborrg** [OwlGamingCommunity/V#45] [Support]
Every time i boot the sever i get this issue, no erros in the server log are shown
any help?
![Desktop Screenshot 2024 07 10 - 17 21 41 24](https://github.com/OwlGamingCommunity/V/assets/175182540/e8b78cb4-91da-479c-89c3-c5f8faaad8d0)

**Comment by YoussifElish** [comment]
contact me on discord 
username : igravy
